### DISCUSSION TITLE
Avatar rating of own uploads showing as "None" - differs from website

### DISCUSSION THREAD

**Discussion by AdeonWriter** [vrcx-team/VRCX#1049] [General]
![image](https://github.com/user-attachments/assets/68f764ce-aa06-4add-88af-e20b7df2a2ca)

**Comment by AdeonWriter** [comment]
Older uploads that haven't been updated in a while still show correct rating in VRCX

**Comment by Kiokuu** [comment]
It seems the problem is on newer avatars due to security variants and the current code for fetching the performanceRating is incorrect relying on the standard variant return of app.getPlatformInfo 

In cases "None" is seen, The security variant tends to have the real performance rating, while the standard variant has 'None' in the performance rating, some avatars have the inverse of this, where standard has a rating and security does not.

Older avatars seemingly have correct performanceRatings in both the standard and security variants.
